docs/release-notes/2.30.0.md

```diff
@@ -0,0 +1,138 @@
+        
+        This dashboard contains panels for data about the Hottest Collections in the MongoDB database.
+
+    - **Experimental MongoDB Collection Details**
+
+        This experimental dashboard provides detailed information about the top collections by document count, size, and document read for MongoDB databases.
+
+    - **Experimental MongoDB Oplog Details**
+
+        This real-time dashboard contains Oplog details such as Recovery Window, Processing Time, Buffer Capacity, and Oplog Operations.
+
+
+### Database as a Service (DBaaS)
+
+Enhanced the user experience by simplifying the configuration and deployment of DBaaS database clusters:
+
+- With PMM 2.30.0, you can create a DB cluster literally at the click of a button. All the fields will be automatically populated with the default value. Click Register, and your DB cluster is created.
+
+- You can easily create a Kubernetes cluster with PMM 2.30.0. When you copy the value of the Kubeconfig file, the Kubernetes Cluster Name gets auto-populated from the contents of the kubeconfig file.
+
+- Starting with PMM 2.23.0, we have simplified the DBaaS configuration and deployment of database clusters by providing suggestions on the k8s cluster name based on the Kubeconfig file.
+
+### Operators support
+
+- PMM now supports Percona Operators MySQL (1.11) and MongoDB (1.12).
+
+- PMM now provides MongoDB v5 deployment support. 
+
+
+
+
+## New Features
+
+- [PMM-9652](https://jira.percona.com/browse/PMM-9652): Dashboard: New experimental dashboards for MongoDB are now available with more MongoDB metrics and data like Hottest collections, Data Size, recovery Window, Processing Time, Buffer Capacity, Oplog Operations, and so on.
+
+- [PMM-10367](https://jira.percona.com/browse/PMM-10367): Dashboard: We have introduced a new experimental Home Dashboard in PMM version 2.30.0. The primary goal of this Home dashboard is to improve the performance while monitoring a large number of nodes or services in PMM. Repeated rows (for a large number of nodes) on the dashboard can take a considerable time to load Home Dashboard UI. So, to address this issue, we have developed a very user-friendly and insightful new experimental Home Dashboard.
+
+- [PMM-10327](https://jira.percona.com/browse/PMM-10327):  You can create a free K8s cluster via Percona Platform Portal and use it for testing DBaaS in PMM.
+    
+    !!! note alert alert-primary ""
+        This feature will be available on https://portal.percona.com/ with the PMM 2.30 GA release.
+
+
+ 
+## Improvements
+
+- [PMM-10340](https://jira.percona.com/browse/PMM-10340): DBaas: Enhanced user experience to simplify DBaaS configuration and deployment of Database clusters with suggestions on k8s cluster name based on k8s config file.
+  
+- [PMM-10368](https://jira.percona.com/browse/PMM-10368): Simpler DBaaS and Percona Platform connection configuration by suggesting the public server address of the PMM server.
```

Comment:
redundancy: "public server address of the PMM server" -> I guess "public address of the PMM server" or "public server address of PMM".